spec/compiler/codegen/const_spec.cr

```diff
@@ -446,4 +468,42 @@ describe "Codegen: const" do
 
     mod.to_s.should_not contain("CONST")
   end
+
+  it "synchronize initialization of contants" do
```

Comment:
```suggestion
  it "synchronize initialization of constants" do
```

Reply:
BTW, I think `synchronize` should have an `s` too.